dataList.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * Formats and lists the contents of each field
+ *
+ * @param array $sets
+ * @return mixed
+ */
+
+function dataList(array $sets): string {
+    // create list as empty string
+    $list = '';
+    foreach ($sets as $set)
+    {
+        $list .= "<h2>" . $set['setName'] . "</h2>";
```

Comment:
Single quotes can be used here too 🧐

Reply:
(Alternatively you could leave the double quotes and use interpolation to remove the concatenation .'s, e.g: `"$list .= "<h2> {$set['setName']} </h2>"; `({} are used to indicate a variable inside double quotes)